client/src/components/utils/index.js

```diff
@@ -0,0 +1,3 @@
+import Loading from './Loading';
+
+export default Loading;
```

Comment:
use the named export for this, as others also will export their components in this file